Sources/Moya/Plugins/AccessTokenPlugin.swift

```diff
@@ -13,15 +13,27 @@ public protocol AccessTokenAuthorizable {
 // MARK: - AuthorizationType
 
 /// An enum representing the header to use with an `AccessTokenPlugin`
-public enum AuthorizationType: String {
+public enum AuthorizationType {
     /// No header.
     case none
 
     /// The `"Basic"` header.
-    case basic = "Basic"
+    case basic
 
     /// The `"Bearer"` header.
-    case bearer = "Bearer"
+    case bearer
+    
+    /// Custom header implementation.
+    case custom(String)
+    
+    public var value: String {
+        
+        switch self {
+        case .none: return ""
+        case .basic, .bearer: return String(describing: self).capitalized
```

Comment:
Can you just use a String literal "Basic" here please? I trust you not to misspell it 😝

Reply:
We have to split it into 2 separate cases. We lost `rawValue` because we have an associated value now. The reason I suggested this is I don't want to rely on `String(describing:)` then `capitalized` to save 1 line of code.

Reply:
I agree, never really trusted `String(describing:)` 😅 2 separate cases seems reasonable 👍